FIX setDate

daisukey623 opened the pull request:
以下修正
・可読性、保守性向上のため、templete内で処理していた日付関数をcomputed内で処理。

daisukey623 committed:
FIX setDate

以下修正
・可読性、保守性向上のため、templete内で処理していた日付関数をcomputed内で処理。

TakehiroTada commented on src/components/Organisms/FeedComments.vue:
```diff
@@ -77,10 +64,18 @@ export default {
-      'post',
-    ]),
+    setDate: function() {
+      return function(item) {
+        return `
+                ${item.createdAt.toDate().getFullYear()}/
+                ${item.createdAt.toDate().getMonth() + 1}/
+                ${item.createdAt.toDate().getDate()}
+                ${item.createdAt.toDate().getHours()}:
+                ${item.createdAt.toDate().getMinutes()}
+              `;
+      };
```
これをやりたい場合は computedじゃなくて methodsにあるべきな気がしてますね
この `item` が参照出来てなくて

>  Error in render: “TypeError: Cannot read property ‘toDate’ of undefined

になってる気がします

TakehiroTada reviewed (changes requested):
コメントしてます

daisukey623 committed:
FIX myPosts

以下修正
・プロフィール内の自分の投稿データをコンポーネントではなくstoreで経由で取得できるようにした

・ beforeRouteUpdateでルーティングの際に同一コンポーネントでもデータ更新されるようにした

daisukey623 committed:
FIX score

・フォロワーのプロフィールページ閲覧時、サイドバーのプロフィールクリック時に、再描画されるように修正。
